ansible/roles/kubernetes-cni/tasks/main.yml

```diff
@@ -1,3 +1,6 @@
 ---
 - include_tasks: calico.yml
   when: kubernetes_cni.plugin == "calico"
+
+- include_tasks: flannel.yml
+  when: kubernetes_cni.plugin == "flannel"
```

Comment:
If there will be support for additional cni plugins in the future, it may make sense to do something like this instead:
```
- fail:
    msg: "CNI plugin {{ kubernetes_cni.plugin }} is not supported"
  when: kubernetes_cni.plugin not in ['calico', 'flannel']

- include_tasks: "{{ kubernetes_cni_plugin }}.yml"
```